Dockerfile

```diff
@@ -22,9 +23,11 @@ RUN apk add --no-cache \
   # Required for img's builds
   pigz=~2.6 \
   jq=~1 \
-  # node.js sites, e.g. pluginsite
-  nodejs=~16 \
-  npm=~8
+  gcompat=~1
```

Comment:
If we’re doing this we may as well just switch to Debian base right?

PR looks great apart from that

Reply:
that would be my vote as well, but I wanted to push what I had so someone else could take a look at it.

Reply:
Note when I tried using this image built locally. Without gcompat node wouldn't run. After adding it,  node ran but wouldn't create directories or files for nom install.

May want to tell the asdf plugin to node build instead of gcompat